scripts/functions.sh

```diff
@@ -164,6 +164,51 @@ function clusterIsReady() {
   fi
 }
 
+function checkClusterVersion() {
+  version=$(${K8S_CLI_BIN} get clusterversion version -o jsonpath='{.status.history[*].version}')
+  if [[ -z "${version}" ]]; then
+    echo "Can't check version since cluster is not OpenShift"
+  else 
+    returnCode=0
+    result=""
+  
+    echo "OpenShift version: $version"
+    if [[ "$command" = "packets" ]]; then
+      compare_versions "$version" 4.16.0
```

Comment:
do you know if `compare_versions` will work with funky versions like nightly / ci and others?

Reply:
Integration tests passed using `OpenShift version: 4.20.0-0.nightly-2025-09-01-101753`

You can find all the logs in the `artifact` folder: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/netobserv_network-observability-cli/373/pull-ci-netobserv-network-observability-cli-main-integration-tests/1962885449894596608/artifacts/integration-tests/netobserv-cli-tests/artifacts/